package.json

```diff
@@ -51,10 +51,10 @@
     },
     "./icu.macro": {
       "types": {
-        "require": "./icu.macro..d.ts",
-        "import": "./icu.macro..d.mts"
+        "require": "./icu.macro.d.ts",
+        "import": "./icu.macro.d.mts"
       },
-      "default": "./icu.macro.js"
+      "default": "./dist/commonjs/icu.macro.js"
```

Comment:
we may streamline this like the other exports... import, require, etc... commonjs and es

Reply:
Since react-i18next was always a JavaScript module and only later shipped with types (but still having javascript code), I would say we stay with JavaScript, until there's a real need to move everything to TypeScript.